pkg/cloudprovider/vsphere/route/config/config_ini_legacy_test.go

```diff
@@ -0,0 +1,53 @@
+/*
+ Copyright 2020 The Kubernetes Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package config
+
+import (
+	"testing"
+)
+
+/*
+	TODO:
+	When the INI based cloud-config is deprecated. This file should be deleted.
+*/
+
+func TestReadINIConfig(t *testing.T) {
+	contents := `
+[Route]
+router-path = /infra/tier-1s/test-router
+
+[NSXT]
+user = admin
```

Comment:
Would it be possible to support `secretName` and `secretNamespace` here in the same way we do for vCenter credentials? I think many users require all credentials are stored in secrets, sometimes the cloud config itself is stored in a ConfigMap.

Reply:
Here's one example https://github.com/kubernetes/cloud-provider-vsphere/blob/22e85632bfe62f9a6afff8d2fc269f80eb9e7c78/pkg/common/config/config_yaml_test.go#L52-L53 

It would require more plumbing on the generic NSX-T package to read the secret from an informer cache

Reply:
Sure, I think we can support secret to store password or certificates. I see there is a `CredentialManager`, let me see if I can leverage it for NSX-T as well.

Reply:
Could I add NSX-T related cert in a following PR?

Reply:
yeah I'm fine with that